addons/website/static/src/js/content/website_root.js

```diff
@@ -187,13 +188,8 @@ var WebsiteRoot = publicRootData.PublicRoot.extend(KeyboardNavigationMixin, {
         document.body.classList.add('o_fullscreen_transition');
         document.body.classList.toggle('o_fullscreen', this.isFullscreen);
         document.body.style.overflowX = 'hidden';
+        this.trigger_up('adjust_iframe');
         let resizing = true;
-        window.requestAnimationFrame(function resizeFunction() {
-            window.dispatchEvent(new Event('resize'));
-            if (resizing) {
-                window.requestAnimationFrame(resizeFunction);
-            }
-        });
         let stopResizing;
         const onTransitionEnd = ev => {
```

Comment:
You removed this code using the `resizing` variable and kept the rest... ?

Reply:
I have removed this code and made a custom event that does the same, I have kept the `resizing` variable as it was used for `stopResizing`